### ISSUE TITLE
[SYSTEM]: Tags doesn't save?

### ISSUE THREAD

**Issue by Ikabodo** [ZotyDev/objects-interactions-fx#9]
### Discord Username

ikabodo

### System

https://github.com/foundryvtt/dnd5e

### Notes on the Support

![image](https://user-images.githubusercontent.com/7766890/232260883-b16912df-8526-4084-88c8-4136865f34b0.png)

This new module looks amazing, but I am having problems getting it to work. I'll do some more deep error searching tomorrow (I probably need to remove other modules as they might be interfering.)

Long story short I'm not getting the module to work at all. No piles are created, and tags doesn't seem to fire. I get the above error when reloading a world and thought it might be relevant. 

If you open the master tag window after a reload nothing appears.

Again, sorry if the info is a bit short for now. I'll get back to you.

**Comment by ZotyDev** [comment]
I've caused this, didn't implemented a section of the code... I'm currently working on a fix.

**Comment by ZotyDev** [comment]
Thanks for reaching out! This bug should be fixed now :D
